api/v1alpha1/experiment_conversion.go

```diff
@@ -0,0 +1,61 @@
+/*
+Copyright 2020 GramLabs, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"github.com/redskyops/redskyops-controller/api/v1beta1"
+	"k8s.io/apimachinery/pkg/conversion"
+	conv "sigs.k8s.io/controller-runtime/pkg/conversion"
+)
+
+var _ conv.Convertible = &Experiment{}
```

Comment:
This makes our types convertible via webhook (once we choose to add one). Using a transient scheme ensures we do not call generated conversion methods inappropriately. Also, the way the generated conversion methods work, as soon as there is a conflict between the objects that cannot be generated, it drops the method and forces you to implement it yourself.